### PR TITLE
feat(core): adjust center peek animation

### PR DESCRIPTION
**Pull request by CatsJuice**

### CODE REVIEW

**Comment by CatsJuice** [comment]
* **#7393** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7393?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @CatsJuice and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/modules/peek-view/view/image-preview/index.tsx [hunk 1]
```diff
@@ -451,7 +451,7 @@ export const ImagePreviewPeekView = (
 ): ReactElement | null => {
   const [blockId, setBlockId] = useState<string | null>(props.blockId);
   const peekView = useService(PeekViewService).peekView;
-  const onClose = peekView.close;
+  const onClose = useCallback(() => peekView.close(), [peekView]);
```
Comment (pengx17) [inline packages/frontend/core/src/modules/peek-view/view/image-preview/index.tsx:454]: Any reasons for this change?

File: packages/frontend/core/src/modules/peek-view/services/peek-view.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { OnEvent, Service } from '@toeverything/infra';
 import { WorkbenchLocationChanged } from '../../workbench/services/workbench';
 import { PeekViewEntity } from '../entities/peek-view';
 
-@OnEvent(WorkbenchLocationChanged, e => e.peekView.close)
+@OnEvent(WorkbenchLocationChanged, e => () => e.peekView.close())
```
Comment (pengx17) [inline packages/frontend/core/src/modules/peek-view/services/peek-view.ts:6]: Care to explain?

File: packages/frontend/core/src/modules/peek-view/view/image-preview/index.tsx (hunk 1 above)
Comment (CatsJuice) [reply packages/frontend/core/src/modules/peek-view/view/image-preview/index.tsx:454]: `peekView.close()` now will accept a parameter, which will cause `MouseEvent` be passed here.

Comment (pengx17) [reply packages/frontend/core/src/modules/peek-view/view/image-preview/index.tsx:454]: I see. Thanks!

**Comment by pengx17** [comment]
image
Looks like performance is OK. The only spike issue happens when drawing canvas elements & toolbar.

**Review by pengx17 — APPROVED** [review]

**Pull request merged**